VideoLocker/src/test/java/org/edx/mobile/test/http/HttpBaseTestCase.java

```diff
@@ -195,7 +197,7 @@ private static boolean urlMatches(String url, String template) {
      * Fetches the mock response from the predefined set in the
      * properties file
      */
-    private static String getMockResponse(String apiPropertyName) throws IOException {
+    protected static String getMockResponse(String apiPropertyName) throws IOException {
         Properties responses = new Properties();
```

Comment:
Might as well. I was going to eventually deprecate this base test case but might as well slowly refactor it.